conf/php-fpm.ini

```diff
@@ -1,2 +0,0 @@
-upload_max_filesize=16M
-post_max_size=16M
```

Comment:
As you remove this file, those 2 lines should be moved to the php-fpm.conf file.

Reply:
Yes, my bad, I just update https://github.com/YunoHost-Apps/concrete5_ynh/pull/3/files#diff-405e07c4665af155b4672de978c8bddeR422